client/src/helpers.js

```diff
@@ -0,0 +1,62 @@
+/** @private */
+class Helpers {
+  /** @private */
+  constructor() {
+    this.listeners = [];
+
+    window.addEventListener(
+      'message',
+      function(evt) {
+        this.listeners
+          .filter(listener => listener.name == evt.data.msg)
+          .map(listener => listener.eventFn(evt, listener.listenerId));
+      }.bind(this)
+    );
+  }
+
+  /**
+   * Registers a post message listener
+   * Don't forget to remove the event listener at the end of
+   * your eventFn if you do not need it anymore.
+   * @private
+   * @param name string event name
```

Comment:
as I remember `@param` should be written as a type like `{string}`, `{function}` etc.

Reply:
Since it is a private function it would not cause any issues, but I agree it should be same as exposed docu.